src/patterns.jl

```diff
@@ -9,9 +9,14 @@ function, which must return a `Matrix{<: AbstractRGB}`.
 abstract type AbstractPattern{T} <: AbstractArray{T, 2} end
 
 function Base.show(io::IO, ::MIME"text/plain", p::AbstractPattern)
+    
     println(io, typeof(p))
-    show(io, mime, image(p))
+    print(io, p)
 end
+
+Base.show(io::IO, p::AbstractPattern) = print(io, image(to_image(p)))
```

Comment:
This should not have `image`, just `to_image`. The output should either be an actual image like Colors.jl has or a matrix style print. Right now it creates a full plot and then prints the Scene structure.

Also the printing for `ImagePattern` still fails.